test/service/test_manifest.py

```diff
@@ -1488,8 +1489,8 @@ def test(*, format: ManifestFormat, fetch: bool, url: Optional[furl] = None):
                 expected_url = object_url
                 expected_url_for_bash = f"'{expected_url}'"
             if format is ManifestFormat.curl:
-                manifest_options = '--location --fail'
-                file_options = '--fail-early --continue-at - --retry 2 --retry-delay 10'
+                manifest_options = ' '.join(CurlManifestGenerator.manifest_options)
+                file_options = ' '.join(CurlManifestGenerator.file_options)
```

Comment:
Using code-under-test to compute the expected value makes the test more or less pointless.